client/src/shared/mockData.js

```diff
@@ -1,22 +1,65 @@
 export const mockShoutData = [
-  {
-    channel_name: 'dev-slack-testing2',
-    date: 'Mon Oct 19 2020',
-    shoutee: {
-      email: 'Bruce.Hartman@techtonic.com',
-      first_name: 'Bruce',
-      last_name: 'Hartman',
-      slack_handle: 'Bruce Hartman',
-      user_id: 'U01BZ777EFJ',
-    },
-    shouter: {
-      email: 'travis.gee@techtonic.com',
-      first_name: 'travis',
-      last_name: 'gee',
-      slack_handle: 'Travis',
-      user_id: 'U01B6CX7L9X',
-    },
-    shoutout: 'Bruce did a thing',
-    values: 'Company value number 1',
-  },
-];
+         {
+           channel_name: "dev-slack-testing2",
+           date: "Fri Oct 16 2020",
+           shoutee: {
+             email: "elliot.mackinnon@techtonic.com",
+             first_name: "elliot",
+             last_name: "mackinnon",
+             slack_handle: "Elliot Mackinnon",
+             user_id: "U01AD22LWCX",
+           },
+           shouter: {
+             email: "travis.gee@techtonic.com",
+             first_name: "travis",
+             last_name: "gee",
+             slack_handle: "Travis",
+             user_id: "U01B6CX7L9X",
+           },
+           shoutout: "For letting us live in his world ",
+           values: "Company value number 1",
+           shoutId: 1,
+         },
+         {
+           channel_name: "dev-slack-testing2",
+           date: "Fri Oct 16 2020",
+           shoutee: {
+             email: "brian.roxbury@techtonic.com",
+             first_name: "brian",
+             last_name: "roxbury",
+             slack_handle: "Brian Roxbury",
+             user_id: "U01B001LJNQ",
+           },
+           shouter: {
+             email: "travis.gee@techtonic.com",
+             first_name: "travis",
+             last_name: "gee",
+             slack_handle: "Travis",
+             user_id: "U01B6CX7L9X",
+           },
+           shoutout: "For making being here today ",
+           values: "Company value number 1",
+           shoutId: 2,
+         },
+         {
+           channel_name: "dev-slack-testing2",
+           date: "Fri Oct 16 2020",
+           shoutee: {
+             email: "brian.roxbury@techtonic.com",
+             first_name: "brian",
+             last_name: "roxbury",
+             slack_handle: "Brian Roxbury",
+             user_id: "U01B001LJNQ",
+           },
+           shouter: {
+             email: "travis.gee@techtonic.com",
+             first_name: "travis",
+             last_name: "gee",
+             slack_handle: "Travis",
+             user_id: "U01B6CX7L9X",
+           },
+           shoutout: "Hi there",
+           values: "Company value number 1",
+           shoutId: 3,
+         },
+       ];
```

Comment:
We should also vary the data a little so that it is apparent when the table is sorting

Reply:
Good call, I tested it a little bit, but I can make a ticket to change the data.

Reply:
We can seed the db as well, so it's up to you if you want to wait or alter the current mock data now